SU-SWS/stanford_profile
# 8.1.10

# READY FOR REVIEW

# Summary
- Use state for the home, 404 and 403 pages since the new default content doesn't consist of node IDs.
- Added field permissions to restrict access to the site url in the site settings.

# Need Review By (Date)
- asap

# Urgency
- Critical

# Steps to Test
1. `composer require su-sws/stanford_profile:dev-home-url-test`
1. do a fresh install `blt drupal:install`
1. Verify the home page doesn't redirect to another path like `/home`

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
